frontend/src/entities/configuration/activated-services-configuration.tsx

```diff
@@ -38,6 +38,18 @@ export type ActivatedServicesConfigurationProps = {
     enabledServices: {[key: string]: boolean};
 };
 
+const generateDescription = () => {
+    return `
+        Activate or deactivate services within MLSpace. Deactivated 
+        services will no longer appear within the MLSpace user interface${!window.env.MANAGE_IAM_ROLES ? `. 
```

Comment:
nit: Since we aren't actually doing anything with the logic here, I would prefer that these ternary operations are positive (remove the !) so it is a bit easier to read.

Reply:
Fair point, will swap them around